properties.schema

```diff
@@ -0,0 +1,161 @@
+{
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "id": "http://jsonschema.net",
+  "$ref": "http://localhost/plugins/content/component/model.schema",
+  "globals": {
+    "ariaRegion": {
+      "type": "string",
+      "required": true,
+      "default": "This is a media component which displays a Vimeo video. Select the play / pause button to watch it.",
+      "inputType": "Text",
+      "validators": []
+    },
+    "transcriptButton": {
```

Comment:
This shouldn't be here... also I think you're missing `skipToTranscript` so maybe change to to that?